README.md

```diff
@@ -9,13 +9,9 @@ the terminal next to where you are already working with `git` and your code.
 
 While in beta, GitHub CLI is available for repos hosted on GitHub.com only. It currently does not support repositories hosted on GitHub Enterprise Server or other hosting providers. We are planning on adding support for GitHub Enterprise Server after GitHub CLI is out of beta (likely towards the end of 2020), and we want to ensure that the API endpoints we use are more widely available for GHES versions that most GitHub customers are on.
 
-## We need your feedback
+## We want your feedback
 
-GitHub CLI is currently in its early development stages, and we're hoping to get feedback from people using it.
-
-If you've installed and used `gh`, we'd love for you to take a short survey here (no more than five minutes): https://forms.gle/umxd3h31c7aMQFKG7
-
-And if you spot bugs or have features that you'd really like to see in `gh`, please check out the [contributing page][]
+We'd love to hear your feedback about `gh`. If you spot bugs or have features that you'd really like to see in `gh`, please open an [issue][] and check out the [contributing page][].
```

Comment:
1. Nit: could the whole phrase “open an issue” be a link so that the verb is included?
2. Does it make sense to tell someone to check out the contributing page after we've already told them to open an issue? I'm thinking this because the contributing page will also instruct people to open an issue, so in the end, we're just telling them to open an issue twice.
3. We're getting lots of of duplicate bug reports/feature requests in a week. Instead of instructing people to open a new issue right away, could we link them to the list of open bugs/feature requests and instruct people to look for existing ones first, and then choose the “new issue” button if they don't find it? I think something like this might help lower the duplicate issue count.

Reply:
@mislav I like that idea. We could add that instruction to `CONTRIBUTING.md` and go back to just linking there first?